internal/cmd/egctl/experimental.go

```diff
@@ -0,0 +1,23 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package egctl
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func NewExperimentalCommand() *cobra.Command {
+
+	experimentalCommand := &cobra.Command{
+		Use:     "experimental",
+		Aliases: []string{"x"},
+		Short:   "Experimental features",
```

Comment:
great! 

can you update the `egctl` doc, `Experimental features do not guarantee compatibility and stability.`

Reply:
sure will do, will also add a user facing `egctl` doc along with this PR once ready